src/templates/csr/manual-multiple/index.svelte

```diff
@@ -0,0 +1,137 @@
+<script lang="ts">
+	import type { GAMVariable } from '$lib/gam';
+	import '$templates/components/fonts/Sans.css';
+	import ManualCard from '$templates/components/ManualCard.svelte';
+	import ManualHeader from '$templates/components/ManualHeader.svelte';
+	import Resizer from '$templates/components/Resizer.svelte';
+	import ToneLogo from '$templates/components/ToneLogo.svelte';
+
+	// export let Title: GAMVariable; // Unused??
```

Comment:
Can we check with Scott Wolfe whether this is ever used? Otherwise let's remove this

Reply:
Scott doesn't deal with the merch ads, just the DAP ones like fabric, the variable doesn't appear in the old template so it can't have been used.